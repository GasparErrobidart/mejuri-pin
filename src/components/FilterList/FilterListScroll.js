import styled              from 'styled-components'

const FilterListScroll = styled.div`
  overflow-y:scroll;
  max-height:50vh;
  border-top: 1px solid ${({theme})=> theme.colors.lightGray};
  @media (min-width:768px){
    max-height:auto;
    overflow-y:visible;
    border:none;
  }
`

export default FilterListScroll
