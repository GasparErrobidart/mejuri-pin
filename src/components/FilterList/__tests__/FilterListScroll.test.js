import '@testing-library/jest-dom'
import 'jest-styled-components'

import React                 from 'react'
import {ThemeProvider}       from 'styled-components'
import { cleanup }           from '@testing-library/react'
import renderer              from 'react-test-renderer'

import theme                 from '../../GlobalStyle/themes/mejuri-pin'

import FilterListScroll   from '../FilterListScroll'


afterEach(cleanup)

test('Renders "FilterListScroll" component', () => {
  const testText        = "FilterListScroll Test Text"
  const component       = renderer.create(
    <ThemeProvider theme={theme}>
      <FilterListScroll>{testText}</FilterListScroll>
    </ThemeProvider>);
  const tree            = component.toJSON()
  expect(tree).toMatchSnapshot();
});
