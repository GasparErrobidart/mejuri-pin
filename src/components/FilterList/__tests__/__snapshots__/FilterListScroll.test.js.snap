// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "FilterListScroll" component 1`] = `
.c0 {
  overflow-y: scroll;
  max-height: 50vh;
  border-top: 1px solid #F3F3F3;
}

@media (min-width:768px) {
  .c0 {
    max-height: auto;
    overflow-y: visible;
    border: none;
  }
}

<div
  className="c0"
>
  FilterListScroll Test Text
</div>
`;
